syntax = "proto3";
package ClientPro;

option cc_generic_services = true;

message LoginRequest {
    string username = 1;
    string password = 2;
}

message ResultCode {
    bool success = 1;
    string errorMsg = 2;
}

message LoginResponse {
    ResultCode status = 1;
    string msg = 2;
}

service ClientServiceRPC {
    rpc Login(LoginRequest) returns(LoginResponse);
}